lib/index.js

```diff
@@ -128,14 +128,15 @@ function parseParameters(value) {
   // eslint-disable-next-line max-params
   function replacer(_, $1, $2, $3, $4) {
     /** @type {MarkerParameterValue} */
-    let value = $2 || $3 || $4 || ''
+
+    let value = $2 ?? $3 ?? $4
```

Comment:
Oh this one would be a breaking change, better to be safe. Not pretty but:

```suggestion
    let value = $2 === undefined ? ($3 === undefined ? $4 : $3) : $2;
```